docs/power-payment-app.md

```diff
@@ -0,0 +1,127 @@
+
+- **Sending Money:** A PowerPay user can send money to another PowerPay user using either a smartphone or a Choronko device. The user initiates the transaction through the frontend client, providing the recipient's details and the desired amount. The backend server processes the transaction, deducts the amount from the sender's account, and updates the recipient's account accordingly.
+
+- **Withdrawing Money:** PowerPay users can withdraw money from their PowerPay accounts using either a smartphone or a Choronko device. The user initiates the withdrawal process through the frontend client, specifying the withdrawal amount and the preferred withdrawal method (e.g., PowerPay kiosk). The backend server processes the request, deducts the amount from the user's account, and facilitates the withdrawal through the appropriate channel.
+
+- **Checking Balance:**  PowerPay users can conveniently check their account balance at any time.  Here's how this works:
+
+1. **User Initiates Check:** The user opens the PowerPay app and navigates to the designated section for checking their balance (e.g., "check balance interface").
+2. **Front-End Request:** The app sends a request through the API to the backend server specifically designed to retrieve the user's account balance.
+3. **Back-End Processing:** The backend server authenticates the user and retrieves the latest balance information from the user database.
+4. **Response to App:** The backend server sends a response back to the app containing the user's current account balance.
+5. **Balance Display:** The app receives the response and updates the user interface to display the retrieved balance information.
+
+
+**see mockup**:
+![reference image](/mockups_wireframes/User_flow_interface.png) 
+![reference image](/mockups_wireframes/Transactions.png)
+
+
+## Error Handling in the PowerPay App
+The PowerPay app strives to provide a smooth user experience by gracefully handling errors that might occur during communication with the back-end server. Here's how the app will address potential issues:
+
+Types of Errors:
+
+- Network Errors: These occur when the user's device loses internet connectivity or cannot establish a connection to the server.
+- Server Errors: These can happen due to issues on the back-end server, such as high load, bugs, or maintenance.
+- API Errors: Errors returned from the back-end APIs due to invalid data formats, unauthorized access attempts, or other API-specific issues.
+
+##Error Handling Strategies:
+
+- Informative Error Messages: The app will display user-friendly error messages that clearly communicate the nature of the problem.for instance:
```

Comment:
Excellent! But don't write "the app will", stay simple and keep "the app displays the error message..."

It keeps more assertive about what we're doing.